pymc/printing.py

```diff
@@ -58,6 +58,9 @@ def str_for_dist(rv: TensorVariable, formatting: str = "plain", include_params:
 def str_for_model(model: Model, formatting: str = "plain", include_params: bool = True) -> str:
     """Make a human-readable string representation of Model, listing all random variables
     and their distributions, optionally including parameter values."""
+    if len(model.unobserved_RVs) + len(model.observed_RVs) + len(model.potentials) == 0:
```

Comment:
Instead of adding this verbose `if statement` you can check if `rv_reprs` below is an empty list before going into the the `if "latex"... else` statement.

Reply:
Good question. That relates to #5579, in the meantime you can perhaps add it to `test_model.py`?